src/main/resources/runners/sigtest/pom.xml

```diff
@@ -127,22 +128,32 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.netbeans.tools</groupId>
+                <!--                <groupId>org.netbeans.tools</groupId>-->
+                <groupId>jakarta.tck</groupId>
                 <artifactId>sigtest-maven-plugin</artifactId>
+                <version>2.2</version>
                 <executions>
-                  <execution>
-                    <id>default-cli</id>
-                    <phase>integration-test</phase>
-                    <goals>
-                      <goal>check</goal>
-                    </goals>
-                  </execution>
+                    <execution>
+                        <id>default-cli</id>
+                        <!--                    <phase>integration-test</phase>-->
+                        <goals>
+                            <!--                      <goal>check</goal>-->
+                            <goal>generate</goal>
+                        </goals>
+                    </execution>
                 </executions>
+                <!--                <configuration>-->
+                <!--                  <action>strictcheck</action>-->
+                <!--                  <failOnError>true</failOnError>-->
+                <!--                  <packages>jakarta.batch.**</packages>-->
+                <!--                  <classes>${project.build.directory}/dependency</classes>-->
+                <!--                </configuration>-->
                 <configuration>
-                  <action>strictcheck</action>
-                  <failOnError>true</failOnError>
-                  <packages>jakarta.batch.**</packages>
-                  <classes>${project.build.directory}/dependency</classes>
+                    <sigfile>${project.build.directory}/jakarta.batch.sig_${project.parent.version}</sigfile>
```

Comment:
This value should be `${project.build.directory}/sigtest-copy/sigtest/sigtest-1.6-batch.standalone.tck.sig-2.1-se11-OpenJDK-J9`.